__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const db = require("../db/connection.js");
+const app = require("../db/app.js");
+const request = require("supertest");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data/index.js");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe('/api/topics', () => {
+    test('GET: 200 status code and sends an array of topics to the client', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response) => {
+            expect(response.body.topics.length).toBe(3)
```

Comment:
Great that you're checking the length of the array to avoid false positives if this test were passed an empty array 🙌